examples/with-amp/pages/index.js

```diff
@@ -0,0 +1,20 @@
+import Head from 'next/head'
+import Link from 'next/prefetch'
```

Comment:
You can use `next/link` here.
Anyway, you don't need it at all. Simply use the `a` tag since there's no any client side next.